lib/pages/laws.dart

```diff
@@ -31,15 +31,115 @@ class Laws extends StatelessWidget {
               children: <Widget>[
                 ListTile(
                   leading: Icon(Icons.label),
```

Comment:
I don't think this icon adds anything to the list visually. I would remove it as well as visually divide one list item from the next